### PR TITLE
Refresh tutorials/namespaces.

### LINKED ISSUE #320: Refresh tutorials/namespaces

### PR DESCRIPTION
**Pull request by jhkrug**
This will close #320

### CODE REVIEW

File: docs/tutorials/namespace-tutorial.md [hunk 1]
```diff
@@ -37,7 +41,14 @@ Name: newnamespace
 ✔️  Namespace created.
 ```
 
-Inspect the details of a given namespace like **creation date**, **applications** and **configurations** with:
+You can inspect the detail of a namespace:
```
Comment (andreas-kupries) [inline docs/tutorials/namespace-tutorial.md:44]:
```suggestion
You can inspect the details of a namespace:
```

File: docs/tutorials/namespace-tutorial.md [hunk 2]
```diff
@@ -56,7 +67,10 @@ Name: newnamespace
 | Configurations |                               |
 ```
 
-By default, `Epinio` creates the namespace `workspace` during installation. If no previous local state existed the `epinio login` command used to get access to the installation also set this namespace as the current namespace. To see this use:
+Epinio creates the default namespace `workspace` at installation.
```
Comment (andreas-kupries) [inline docs/tutorials/namespace-tutorial.md:70]:
```suggestion
Epinio creates the default namespace `workspace` at installation time.
```
Maybe?

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
small things

File: docs/tutorials/namespace-tutorial.md (hunk 2 above)
Comment (jhkrug) [reply docs/tutorials/namespace-tutorial.md:70]: I think it's clear that the 'time' is implied from the context. It's clear that a place is not involved as in:

God creates the garden at Eden.

But sure, either is fine.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**